### ISSUE TITLE
Feature Suggestion: Be able to search for "indirectly linked, but not directly linked"

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#2031]
## Feature suggestion:
Be able to search for "indirectly linked, but not directly linked into a given language, to make it easier for members who are searching for sentences that could be linked.

## Related comment by alanfgh https://github.com/Tatoeba/tatoeba2/issues/1980#issuecomment-562119881 that helps clarify this suggestion.

The idea is that indirectly linked sentences are often good candidates for direct linking, so a search that finds indirectly linked sentences but excludes directly linked sentences would be an efficient way of bringing up good sentence pairs to link directly.

**Comment by Inego** [comment]
Hasn't this feature already been there for a long time?

**Comment by ckjpn** [comment]
> Hasn't this feature already been there for a long time?

No.  What's possibly now is to show sentences that are indirectly linked, but this search also show sentences that have direct links.

What I'm suggesting it to not show all the sentences that have direct links.  This would allow focusing on finding potentially linkable sentences into languages that don't have any direct link to the sentences in the results.

**Comment by Inego** [comment]
Suppose we have a sentence A and four indirect translations (in the same language) B1 ... B4.
One day someone comes and links the first one, B1.
With the filtering logic you describe, you won't be able to find the still unlinked B2, B3 and B4 any more, only because of that already linked B1.
I wonder what the particular scenario this filtering was useful for could be.

**Comment by Inego** [comment]
As far as I understand, currently the link filter works as follows:
* "Any" = "Has at least one direct or indirect translation in the target language"
* "Direct" = "Has at least one direct translation in the target language"
* "Indirect" = "Has at least one indirect translation in the target language"

Technically, I believe, it's easy to add a couple more filters:
* "Has at least one indirect translation, but no direct translations"
* "Has at least one direct translation, but no indirect translations"

**Comment by ckjpn** [comment]
> "Has at least one indirect translation, but no direct translations"

This is the one I mean.

**Comment by Poulpisator** [comment]
Is the best option not to decoupled the filtering methods as Inego suggested (I think) in #1980 ? 

The problem is that right now "Limit to" and "Exclude" are parts of the same filter. If we can simply do "Limit to" having indirect translation and "Exclude" having direct translations together, the problem is solved, I think, and there's no need to add anything. Then, each is free to use the method he wants to add links and 
> you won't be able to find the still unlinked B2, B3 and B4 any more, only because of that already linked B1.

will not be a problem.

**Comment by Inego** [comment]
This could be achieved, if those "Limit to ..." and "Exclude ..." filters could be stacked, yes.

**Comment by ckjpn** [comment]
This is a duplicate of an issue submitted in 2016.
https://github.com/Tatoeba/tatoeba2/issues/1360

Since there seems to be more discussion on this one, maybe it's the older issue that should be closed.  I'm not sure, so I'll let TRANG decide.

**Comment by ckjpn** [comment]
Related Wall Post by Aiji

https://tatoeba.org/en/wall/show_message/39021#message_39021
✨ Sentences with indirect translations and no direct translations ✨
I’ve made lists of sentences in language A having indirect translations but no direct translations in language B. ....
